book/survival.qmd

```diff
@@ -212,79 +244,85 @@ This data would be collected on a yearly scale and over three years may look som
-Truncation on the other hand often leads to observations not entering the data set (at least not at time 0).
+While censored observations have incomplete information about the time-to-event, they are still part of the data set.
+Whereas truncation leads to observations not entering the data set (at least not at time 0).
 This will usually introduce bias that needs to be accounted for.
 
-#### Left-truncation {.unnumbered .unlisted}
+### Left-truncation {.unnumbered .unlisted}
 
 Left-truncation often occurs when study participation is conditional on the occurence of another event.
-This is best explained through examples:
+By example, consider a study from the 18th century [@brostrom.influence.1987], when childhood and maternal mortality were relatively high.
+The goal of the study was to establish the effect of a mother's death on the survival of the infant.
+Since each death was reported to the authorities, an infant was added to the study if and when their mother died.
+To create a matched cohort, two other infants, whose mothers were alive, were matched into the study based on their age and other relevant features.
+Thus, groups of three infants within the study had identical features except for the status of the mother (alive or dead).
+Because of the study design, infants who died before their mothers could never enter into the study. A mother's death is thus refered to as left-truncation event and the infant's age at time of inclusion into the study is refered to as left-truncation time.
 
-In a study  from the 18th century @brostrom.influence.1987, when childhood and maternal mortality were relatively high, data was collected on infant survival during the first year after birth based on whether the mother was alive.
-Data was collected by adding an infant to the study when their mother died, then two other infants, whose mothers were alive, were matched into the study based on their age and features.
-The age of the infant at entrance into the study therefore signifies a *left-truncation event*, as infants who die before their mothers never enter the data and the *left-truncation time* is given by the infants' age at which the mother died.
+More formally, let $t^L_i$ the subject-specific left-truncation time.
+Then we only observe subjects with $y_i > t^L_i$ and subjects with $y_i < t^L_i$ never enter the data.
 
-More formally, let $T^l_i$ the subject-specific left truncation time.
-Then we only observe subjects with $Y_i > T^l_i$ or $C_i > T^l_i$.
-Subjects with $Y_i < T^l_i$ never enter the data.
+This is illustrated in @fig-left-truncation.
+Continuing with the example above, say Infant 1 dies at $t_1$, while the mother dies at some later timepoint $t_1^L$, thefore Infant 1 never enters the study. 
+The mother of Infant 2 dies at $t_2^L$, at which point the infant is included in the study and experiences an event at $t_2$.
+Finally, say Infant 3 enters the study at $t_3^L$ and is censored at 365 days when the study ends.
```

Comment:
As above - the visualisation is great but this doesn't explain what the problem is or what we should do about it